src/RepairManagementAPI/Commands/RejectOrder.cs

```diff
@@ -0,0 +1,14 @@
+public class RejectOrder
+{
+    public Guid MessageId { get; set; }
+    public Guid RepairOrderId { get; set; }
+    public string RejectReason { get; set; }
+    public DateTime RejectedAt { get; set; }
+
+    public RejectOrder(Guid messageId, Guid repairOrderId, string rejectReason)
+    {
+        MessageId = messageId;
+        RepairOrderId = repairOrderId;
+        RejectReason = rejectReason;
```

Comment:
moet de rejectedAt niet met een new DateTime() in de constructor gezet worden ook?

Reply:
Done thanks.